### PR TITLE
Skip validation in ClinVar submission via API

### LINKED ISSUE #3947: Remove validation buttons in ClinVar API submission form
At the same time make sure that any error returned from the submission process is clearly shown in Scout (to fix the submission and resubmit it correclty)

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR changes a functionality:
- Removes the validation step from the ClinVar submission via API - fix #3947 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. **Note that in the code I have replaced the prod submission url with the test one, so it's fine to test any submission for the purpose of this PR**
2. Locally or on test server, prepare a ClinVar submission containing one or more variants and casedata
3. Click directly on "Submit to ClinVar" even if your submission has no associated Submission ID 
4. The submission should be successful - and an official submission ID (SUBnnnnnn) should be saved in database for it
5. Alternatively the submission process should fail, but it should return the errors that occurred

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR DN

### CODE REVIEW

File: scout/constants/clinvar.py
Comment (northwestwitch) [inline scout/constants/clinvar.py:1]: This change is for the sake of testing only, it should be reverted immediately before merging this PR!

File: tests/server/blueprints/clinvar/test_clinvar_views.py
Comment (dnil) [inline tests/server/blueprints/clinvar/test_clinvar_views.py:151]: 🏅

**Review by dnil — APPROVED** [review]
Looks good to me! 

While testing, I noticed that one can hit "submit" also on closed submissions, which does not open the actual modal, only dims the background. In contrast to a submitted submission, which can not be submitted. Upon checking why, it seems the page with the modal is in a little bit of a mess with forms dubiously without content and a couple of mismatched divs. But, this is not the topic of this PR, so feel very free to merge this when satisfied, and we can return with some html fixes later.

File: scout/server/blueprints/clinvar/controllers.py
Comment (northwestwitch) [reply scout/server/blueprints/clinvar/controllers.py:426]: Hehe I was under the impression that it was an overkill! Perhaps I'll make it overwrite on the clinvar id and that's it

File: scout/server/blueprints/clinvar/templates/clinvar/clinvar_howto.html
Comment (northwestwitch) [reply scout/server/blueprints/clinvar/templates/clinvar/clinvar_howto.html:16]: Good idea, I'll change it like this:

image

**Comment by northwestwitch** [comment]
> While testing, I noticed that one can hit "submit" also on closed submissions, which does not open the actual modal, only dims the background. In contrast to a submitted submission, which can not be submitted. Upon checking why, it seems the page with the modal is in a little bit of a mess with forms dubiously without content and a couple of mismatched divs. But, this is not the topic of this PR, so feel very free to merge this when satisfied, and we can return with some html fixes later.

I don't want to touch it again, so I'll fix this thing as well! Nice that you noticed it!

**Comment by northwestwitch** [comment]
> > While testing, I noticed that one can hit "submit" also on closed submissions, which does not open the actual modal, only dims the background. In contrast to a submitted submission, which can not be submitted. Upon checking why, it seems the page with the modal is in a little bit of a mess with forms dubiously without content and a couple of mismatched divs. But, this is not the topic of this PR, so feel very free to merge this when satisfied, and we can return with some html fixes later.
> 
> I don't want to touch it again, so I'll fix this thing as well! Nice that you noticed it!

Fixed:

image

**Pull request merged**
